src/system/api/rqidApi.ts

```diff
@@ -230,6 +270,27 @@ export class Rqid {
     document?.sheet?.render(true, { focus: true });
   }
 
+  /**
+   * Given a Document, set its rqid to the default rqid and the supplied language. 
+   * Returns the rqid flag
+   */
+  public static async setDefaultRqid(document: Document<any, any>, lang: string = "en", priority: number = 0): Promise<any> {
```

Comment:
@wake42  this was related to my attempt to set rqids on existing skills in the starter set which I abandoned.  I thought the method was handy though since you can use it from a macro or the console to set rqids on an entire folder of things in the world at once.  I was also thinking maybe something like the following would be helpful as well.  Just means we don't have to remember all the key names. 

```ts
public static async setRqid(document: Document<any, any>, rqid: string, lang: string = "en", priority: number = 0): Promise<any> { ....
```

Reply:
Maybe even a "getRqid(document....." would be helpful for macro writers?

Reply:
Yes I think a `setRqid` function would be a good addition to the api. And this setDefaultRqid could use that one internally.

Reply:
Added `setRqid`